streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -144,7 +144,7 @@ public DebeziumArgs() {
 		public String jdbcDriverClass = "com.mysql.cj.jdbc.Driver";
 		
 		@Parameter(names = { "--jdbcUrl" }, description = "JDBC URL input")
-		public String jdbcUrlInput = "jdbc:mysql://localhost:3308/openmrs";
+		public String jdbcUrlInput = "jdbc:mysql://localhost:3306/openmrs";
 		
```

Comment:
ya , it should point to the default used by the openmrs test instance

Reply:
thank you for  fixing  this